services/management/commands/turku_service_import/accessibility.py

```diff
@@ -0,0 +1,148 @@
+from munigeo.importer.sync import ModelSyncher
+
+from services.management.commands.turku_service_import.utils import get_ar_resource, get_ar_servicepoint_resource, \
+    get_ar_servicepoint_accessibility_resource
+from services.models import Unit, AccessibilityVariable, UnitAccessibilityProperty
+
+VERBOSITY = False
+LOGGER = None
```

Comment:
These don't seem to be used anywhere.

Reply:
Damnit, relics from the Helsinki code again, thought I had managed to get those out this time 😄